misk/web/@misk/common/src/externals.ts

```diff
@@ -28,7 +26,10 @@ export default makeExternals({
 function makeExternals(inExternals: IExternal) : IExternal {
   const outExternals: IExternal = {}
   Object.keys(inExternals).forEach((name, index) => {
-    outExternals[name] = inExternals.hasOwnProperty(name) ? (Array.isArray(inExternals[name]) ? (inExternals[name] as string[]).join("") : inExternals[name]) : name
-  })
+    outExternals[name] = inExternals.hasOwnProperty(name) ? 
+      (Array.isArray(inExternals[name]) ? 
+        (inExternals[name] as string[]).join("") : inExternals[name]
```

Comment:
Serializing all the externals... this is an interesting piece of code, maybe we can add a comment about why this is happening.

Reply:
Why is this happening?

Reply:
Webpack requires externals in a specific object format, this function produces an object to that specification. Currently that involves concatenating arrays of scoped packages (ie. `[Blueprint, Core] -> "BlueprintCore"). Later that may involve declaring specific strings for the different module frameworks (amd, commonjs, commonjs2, root).

Reply:
This belongs in a comment "Webpack requires externals in a specific object format, this function produces an object to that specification. Currently that involves concatenating arrays of scoped packages (ie. `[Blueprint, Core] -> "BlueprintCore")."

Reply:
Documentation was added to the function:

```
/**
 * 
 * @param inExternals : IExternal
 * 
 * Create Webpack compatible externals object with following modifications
 * - Concatenate scoped package arrays into single strings
 * 
 * Todo
 * - Provide distinct package strings for amd, commonjs, commonjs2, root if necessary
 */
```